ProjectShop/UserApp/models.py

```diff
@@ -5,7 +5,7 @@
 from django.utils.translation import ugettext_lazy as _
 
 from phonenumber_field.modelfields import PhoneNumberField
-from .managers import UserManager
+from UserApp.managers import UserManager
```

Comment:
According to PEP 8:
Imports should be grouped in the following order:
1. Standard library imports.
2. Related third-party imports.
3. Local application/library-specific imports.
You should put a blank line between each group of imports.

Reply:
Done